test/server/cards/03_TWI/units/ProvidenceDestroyer.spec.ts

```diff
@@ -0,0 +1,59 @@
+describe('Providence Destroyer', function() {
+    integration(function(contextRef) {
+        describe('Providence Destroyer\'s ability', function () {
+            it('should give a enemy space unit -2/-2 for the phase', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['droideka-security'],
+                        spaceArena: ['providence-destroyer', 'system-patrol-craft']
+                    },
+                    player2: {
+                        groundArena: ['pyke-sentinel'],
+                        spaceArena: ['consortium-starviper']
+                    }
+                });
+
+                const { context } = contextRef;
+
+
```

Comment:
nit: two empty spaces, also in the other test below

Reply:
fixed